protobuf-maven-plugin/src/it/gh-438-incremental-compilation-no-changes/src/main/protobuf/org/example/user.proto

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (C) 2023 - 2024, Ashley Scopes.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package org.example;
+
+option java_multiple_files = true;
+option java_package = "org.example";
+
+import "org/example/avatar.proto";
+
+message User {
```

Comment:
- [ ] Add java file that references the generated code to catch issues with source registration order.